### PR TITLE
Updated documentation to match new functionality

### PR DESCRIPTION
**Pull request by BradleyClifford**

### CODE REVIEW

File: apis/02-Product Management/03-Property-API-(Contact Expedia Before Adopting)/02-supported-features.md [hunk 1]
```diff
@@ -13,12 +13,13 @@ The following sections indicate specific use cases that are supported or not sup
...
-| Expedia Collect-only Properties | Hotel Collect-only Properties |
-| ExpediaTravelerPreference Properties (offering both Expedia Collect and Hotel Collect products) | *Changing a property's distribution model via API |
+| Supported |
+| --------- |
+| Expedia Collect-only Properties |
+| Hotel Collect-only Properties |
+| ExpediaTravelerPreference Properties (offering both Expedia Collect and Hotel Collect products) |
 
-Property API supports the onboarding of new properties that offer either Expedia Collect products only OR offers both Expedia Collect and Hotel Collect products.  *Contact your account manager if you wish to support changing a property's distribution model via the API.   
+Property API supports the onboarding of new properties that offer Expedia Collect products only OR Hotel Collect products only OR offers both Expedia Collect and Hotel Collect products.
```
Comment (a-jomills) [inline apis/02-Product Management/03-Property-API-(Contact Expedia Before Adopting)/02-supported-features.md]: Maybe use a ',' here? Perhaps something like:

"properties that offer only products of Expedia Collect, Hotel Collect, OR both Expedia Collect and Hotel Collect. "

File: apis/02-Product Management/03-Property-API-(Contact Expedia Before Adopting)/02-supported-features.md [hunk 2]
```diff
@@ -13,12 +13,13 @@ The following sections indicate specific use cases that are supported or not sup
...
-| Expedia Collect-only Properties | Hotel Collect-only Properties |
-| ExpediaTravelerPreference Properties (offering both Expedia Collect and Hotel Collect products) | *Changing a property's distribution model via API |
+| Supported |
+| --------- |
+| Expedia Collect-only Properties |
+| Hotel Collect-only Properties |
+| ExpediaTravelerPreference Properties (offering both Expedia Collect and Hotel Collect products) |
 
-Property API supports the onboarding of new properties that offer either Expedia Collect products only OR offers both Expedia Collect and Hotel Collect products.  *Contact your account manager if you wish to support changing a property's distribution model via the API.   
+Property API supports the onboarding of new properties that offer only products of Expedia Collect, Hotel Collect, OR both Expedia Collect and Hotel Collect.
```
Comment (jgorman-exp) [inline apis/02-Product Management/03-Property-API-(Contact Expedia Before Adopting)/02-supported-features.md]: "...products of Expedia Collect..." sounds odd to me.  How about "...that offer Expedia Collect, Hotel Collect, OR both Expedia Collect and Hotel Collect products."

File: apis/02-Product Management/03-Property-API-(Contact Expedia Before Adopting)/02-supported-features.md [hunk 3]
```diff
@@ -13,12 +13,13 @@ The following sections indicate specific use cases that are supported or not sup
 
 ## Distribution Model
 
-| Supported | Not Supported |
-| --------- | ------------- |
-| Expedia Collect-only Properties | Hotel Collect-only Properties |
-| ExpediaTravelerPreference Properties (offering both Expedia Collect and Hotel Collect products) | *Changing a property's distribution model via API |
```
Comment (jgorman-exp) [inline apis/02-Product Management/03-Property-API-(Contact Expedia Before Adopting)/02-supported-features.md]: We don’t support changing the distribution model via the API, do we?

Comment (BradleyClifford) [inline apis/02-Product Management/03-Property-API-(Contact Expedia Before Adopting)/02-supported-features.md]: I may have misunderstood what this meant, I can add it back as not supported along with the note listed below.

**Pull request merged**